CONTRIBUTING.md

```diff
@@ -12,6 +12,9 @@ When opening new issues or commenting on existing issues on this repository
 please make sure discussions are related to concrete technical issues with the
 Node.js software.
 
+Go to https://github.com/nodejs/node/issues/new and click the 'New Issue' button
+and [fill out the form](.github/ISSUE_TEMPLATE.md).
```

Comment:
This link is misleading. The text implies that you will be taken to a form. Instead, it will take you to a markdown file. I think this entire paragraph should be removed.